proto/panacea/datapool/v2/query.proto

```diff
@@ -36,6 +36,11 @@ service Query {
     option (google.api.http).get = "/panacea/datapool/v2/data_validation_certificates/pool/{pool_id}/round/{round}";
   }
 
+  // DataPassRedeemReceipt returns DataPassRedeemReceipt
+  rpc DataPassRedeemReceipt(QueryDataPassRedeemReceiptRequest) returns (QueryDataPassRedeemReceiptResponse) {
+    option (google.api.http).get = "/panacea/datapool/v2/data_pass_redeem_receipt/pool/{pool_id}/round/{round}/nft/{nft_id}";
```

Comment:
IMHO, DataPass is a sub-concept of a pool, so it would be better to define a URI as follows.
```
/panacea/datapool/v2/pools/{pool_id}/rounds/{round}/data_passess/{nft_id}/redeem_receipt
```
ref: https://restfulapi.net/resource-naming/

It would be good to change other queries similarly.
What are other people's opinions?
@youngjoon-lee @hansol-medi @inchori

Reply:
Agreed 👍

Reply:
I also agree with you so that I'll change the URI of `DataPassRedeemReceipts` in #321 .

Reply:
Looking back at this document, it says to use hyphens instead of underscores.
So it would be good to change our URL to hyphen as well.
```
/panacea/datapool/v2/pools/{pool_id}/rounds/{round}/data-passess/{nft_id}/redeem-receipt
```

Reply:
OK, I'll just leave these comments for now.